### PR TITLE
[ADD] Added widget on topbar to switch company easily if logged in us…

### PR DESCRIPTION
**Pull request by hmo-odoo**
…er has multicompany rights and more than one allowed companies

Task: https://www.odoo.com/web#id=12374&view_type=form&model=project.task&menu_id=3942&action=327

Pad: https://pad.odoo.com/p/r.rXPVFx60UwdLf8R9

### CODE REVIEW

**Comment by jholze** [comment]
Hm, the UX of this integration is not great. Why not just add a simple drop down ? Why should i select and save ?

**Comment by jholze** [comment]
We did a similar feature for the website builder --> 
bildschirmfoto 2015-06-26 um 11 59 09

File: addons/web/static/src/js/widgets/switch_company.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+var _t = core._t;
+
+var SwitchCompany = Widget.extend({
+    template: 'SwitchCompany',
+    start: function() {
+        var self = this;
+        new Model('res.users').call('user_systray_info').then(function(res) {
+            if (res.company) {
+                self.$el.toggleClass("hidden").on('click', function(ev) {
```
Comment (rga-odoo) [inline addons/web/static/src/js/widgets/switch_company.js]: `removeClass` instead of `toggleClass`

Comment (dja-odoo) [inline addons/web/static/src/js/widgets/switch_company.js]: Done.

**Comment by hmo-odoo** [comment]
due to spec change, closed  this PR

**Pull request closed without merging**